connector/pom.xml

```diff
@@ -10,6 +10,10 @@
     </parent>
     <artifactId>connector</artifactId>
 
+    <properties>
+        <netty.version>4.1.56.Final</netty.version>
```

Comment:
This version is outdated now. The newest version is 4.1.58.Final

Reply:
Thanks for the heads-up!

Reply:
https://github.com/netty/netty/releases/tag/netty-4.1.59.Final
This is now 4.0.59.Final :)